cmd/protoc-gen-go-grpc/README.md

```diff
@@ -0,0 +1,15 @@
+# protoc-gen-go-grpc
+
+This tool generates Go language bindings of `service`s in protobuf definition
+files for gRPC.  For usage information, please see our [quick start
+guide](https://grpc.io/docs/languages/go/quickstart/).
```

Comment:
Quick start doesn't have examples for using this plugin (yet?)

Even if it does, it would be helpful to have a copy-paste ready command here?
Or point to https://github.com/grpc/grpc-go/blob/master/regenerate.sh#L60

Reply:
Documentation is very much a work in progress.  Let's make incremental progress on it until we eventually release the tool at 1.0.